releasenotes/notes/file-delete-4ca0baa574907813.yaml

```
@@ -0,0 +1,5 @@
+---
+
+features:
+  - |
+    Added `file_delete` method to delete experiment artifacts.
```

This also needs to be in sphinx format.
```suggestion
    Added the :meth:`~IBMExperimentService.file_delete` method to delete experiment artifacts.
```